examples/accordion/accordion.html

```diff
@@ -38,30 +38,17 @@ <h2 id="ex_label">Example</h2>
     <div role="separator" id="ex_start_sep" aria-labelledby="ex_start_sep ex_label" aria-label="Start of"></div>
     <div id="coding-arena">
       <div class="demo-block">
-        <!-- Accordion Configuration Options
```

Comment:
Was this accidental removal? We do not want to remove these config options or their documentation.